lib/lf/assembly/coomatrix.cc

```diff
@@ -0,0 +1,20 @@
+/***************************************************************************
```

Comment:
I think this file is not needed?

Reply:
I have made it habit to add a .cc file for every .h file. Sometimes needed later. Does not do any damage.